spec/lib/fields/deferred_spec.rb

```diff
@@ -20,7 +20,7 @@
         deferred.permitted_attribute(:foo)
 
         expect(Administrate::Field::String).
-          to have_received(:permitted_attribute).with(:foo)
+          to have_received(:permitted_attribute).with(:foo, {})
```

Comment:
If the method is providing a default (`nil`), is it valuable for us to be providing an empty hash here?

Reply:
Yes, however in that particular case we are testing this method on String field in scope of Deferred attribute, in `deferred.rb` the options that we pass are actually initialized with `{}` by default